StandardLibrary/codebuild/release-python/validate.yml

```diff
@@ -0,0 +1,39 @@
+version: 0.2
+
+# Validate that the released artifact is usable
+# by installing it from PyPI
+# then running tests from source against the published artifact.
+
+phases:
+  install:
+    commands:
+      - pip install "tox < 4.0"
+    runtime-versions:
+      python: latest
+  pre_build:
+    commands:
+      # Get Dafny
+      - export dafnyVersion=$(grep '^dafnyVersion=' project.properties | cut -d '=' -f 2)
+      - cd ..
+      - curl https://github.com/dafny-lang/dafny/releases/download/v$dafnyVersion/dafny-$dafnyVersion-x64-ubuntu-20.04.zip  -L -o dafny.zip
+      - unzip -qq dafny.zip && rm dafny.zip
+      - export PATH="$PWD/dafny:$PATH"
+      # Switch back to the main directory
+      - cd aws-cryptographic-material-providers-library
+      # Install test dependencies
+      - pyenv install --skip-existing 3.11
+      - pyenv local 3.11
+      - pip install "tox < 4.0"
+      - pip install pytest
+      # Install the published artifact from PyPI.
+      # The tests will use the installed package, not the local code.
+      # The actual test code must be from local, since we don't publish tests.
+      - pip install aws-cryptography-internal-standard-library==$VERSION
+  build:
+    commands:
+      - NUM_RETRIES=3
+      - cd StandardLibrary
+      # Transpile code (for tests)
+      - make transpile_python
```

Comment:
I think you only want to transpile the tests, no?

Reply:
Yes, but no language is currently set up only transpile tests.
The `ENABLE_EXTERN_PROCESSING` processing flag would need to be refactored for all languages.
If there were an easy `transpile_only_tests`, I'd use that, but it doesn't exist right now.
That said, the transpiled source isn't used. It would need to be installed with `pip install` to be used.

Reply:
Chatted offline -- plan is to commit dtr files on release.
(This is very similar to our plan to commit generated Go code on releases.)
This is in PR at https://github.com/aws/aws-cryptographic-material-providers-library/pull/735

Reply:
I needed a new Smithy-Dafny target for Python: https://github.com/smithy-lang/smithy-dafny/pull/589